Zend/tests/return_types/019.phpt

```diff
@@ -1,10 +1,10 @@
 --TEST--
-__clone cannot declare a return type
+__clone can only declare void return
 --FILE--
 <?php
 
 class Foo {
     function __clone() : Foo {}
 }
 --EXPECTF--
-Fatal error: %s::%s() cannot declare a return type in %s on line %d
+Fatal error: Method Foo::__clone() must return void in %s on line %d
```

Comment:
I find this confusing. Functions don't return void. Instead they have return type information of void. How about:

```
Fatal error: Method Foo::__clone() only void return type allowed in %s on line %d
```

For this, and the other places where 'return void' is mentioned.

Reply:
@kocsismate You're the "wording" guru, what do you think? I liked Danack's idea.

Reply:
@carusogabriel It's an overstatement :)

My suggestions in https://github.com/php/php-src/pull/4177 were:
- `%s::%s(): Return type can only be void`
- `%s::%s(): Return type must be void when declared`

But I'd suggest that you ask around for more, and better informed opinions in chat ;)

Reply:
By the way, I think you should be consistent with @kocsismate [consistent error messages' PR](https://github.com/php/php-src/pull/5066/files). For example, use `Foo::__clone(): Must return void` instead of `Method Foo::__clone() must return void`.

Reply:
@moliata Just a small correction: https://github.com/php/php-src/pull/5625 is the PR where the return type error message format was changed and https://github.com/php/php-src/pull/5211 where the argument error messages were changed. :) But I agree with you that magic methods should follow these formats as much as possible. :)